regression/strings-smoke-tests/java_long_to_string/Test1.java

```diff
@@ -0,0 +1,9 @@
+public class Test1
+{
+    public static void main()
+    {
+        String s = Long.toString(12);
+        assert(s.equals("12"));
+        assert(!s.equals("12"));
```

Comment:
Having both assertions in the same branch can be a problem because CBMC will assume the result of the first one is true when we get to the second one, so it would be trivially false in that case.
You can add an `if` with a boolean parameter that can be an argument of the function.